lib/infra.js

```diff
@@ -49,4 +50,14 @@ module.exports = async (dir, cmd) => {
   ]);
 
   console.log(config);
+
+  execa('kops', [
+    'create',
+    'cluster',
+    `--name=${config.clusterDomain}`,
+    '--state=s3://cxcloud-kops-test',
+    `--zones=${config.zone}`,
```

Comment:
any purpose for different quotation marks?

Reply:
@DealWithMeal those are template tags (interpolation)